CHANGELOG.md

```diff
@@ -10,9 +10,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Fixed
 - Styling issues in `CheckboxSelect` when `isValueHidden` prop is set to `true`
+- Styling some states of the button
 
 ### Refactored
 - Give `label` element in `Checkbox` and `StyledToggle` a classname to override `.styled-atlaskit-form label` in local dev
+- Icons synchronization with the Figma design
```

Comment:
for the future PR, please make sure your local version of atlaskit/select is the same as in package.json because it affects CheckboxSelect styling